src/OrchardCore.Modules/OrchardCore.Autoroute/Settings/AutoroutePartSettingsDisplayDriver.cs

```diff
@@ -21,7 +21,7 @@ public AutoroutePartSettingsDisplayDriver(ILiquidTemplateManager templateManager
             T = localizer;
         }
 
-        public IStringLocalizer T { get; private set; }
+        public IStringLocalizer T { get; set; }
```

Comment:
Why it's setter?

Reply:
Maybe you didn't see my comment that you can see if you open below your `1 change requested` and go to `See review`. Here i copy pasted this comment. I agree we don't need this setter and **it can be removed in many other places**.

But you commented that you will do a cleanup PR for this, yes i think it is better because it is related to other files that are not changed here. Then  see my below comment where i also talk about `IStringLocalizer` and `IHtmlLocalizer` that could be renamed too and so on. 

So, here i just give you more infos and let you do it as you want in a separate PR.

---

Yes, it's only set in the ctor and only for internal usage so we don't need a setter. I think we made it a property just to use an Uppercase `T` without breaking the naming convention.

I already removed in some places the `private` access modifier, not sure it's good but it is defined like this in most other places.

    public IStringLocalizer T { get; set; }

Oh yes, maybe you meant to remove the setter completely which makes more sense.

    public IStringLocalizer T { get; }

Or just this but that breaks the naming convention, but it doesn't hurt me, i already saw it in the aspnetcore code.

    private IStringLocalizer T;

**So i let you cleanup this as you want in another PR ;)**

But but but, maybe this is not the end of the story. As i remember for all `IStringLocalizer` it would be better to use `S` in place of `T`. And for all `IHtmlLocalizer` it would be better to use `H`.

Reply:
Okay, finally, as i am on thsi i will remove all setters, but without renaming the localizers ;)

Reply:
Just leave it to me you can rebase on dev later

+1 for `private IStringLocalizer T;` coz removing a property for any reason later will be a breaking change

Reply:
It will be better if we add `readonly` to the field :)

Reply:
Too late, i changed it everywhere (88 files) it blowed up my PR, i hate you ;)

Reply:
OMG, I just pushed a PR #4750, I told you leave it to me 😄 I know it blows

Reply:
no problem, i will let you a comment on your new PR